### ISSUE TITLE
DGC warmup 

### ISSUE THREAD

**Issue by yimjinkyu1** [sands-lab/grace#15]
I know DGC algorithm used warm-up training to exponentially increase the gradient sparsity.

Do you imply warm-up algorithm in your code ?

**Comment by hangxu0304** [comment]
No. However, you can change the sparsity dynamically in your training script by calling:
```python
for i in range(epochs):
  grc.compressor.compress_ratio = 0.01*pow(2,i)
```

**Comment by yimjinkyu1** [comment]
Thank you,

I have also another question. 

I think DGC algorithm deploy momentum correction algorithm.
Is it right your code has this algorithm in DGC code ?

The figures for improving the imagenet model speed of DGC original papers are higher than your paper indicators.

**Comment by hangxu0304** [comment]
Yes. The momentum correction is implemented in grace_dl/torch/memory/dgc.py .
I think the hyper-parameters used in our experiment are a bit different from the original author's, e.g. batch size, number of nodes, learning rate, whether apply compression only on high rank gradients, etc. Please refer to the original author's repo for an exact reproduction.
